### PR TITLE
Misc stream blocks

### PR DESCRIPTION
**Pull request by ncorgan**

### CODE REVIEW

File: stream/Repeat.cpp [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        const auto elemsIn = this->workInfo().minInElements;
+        if(0 == elemsIn)
+        {
+            return;
+        }
+
+        auto input = this->input(0);
+        auto output = this->output(0);
+
+        Pothos::BufferChunk outputChunk(output->dtype(), elemsIn*_repeatCount);
```
Comment (guruofquality) [inline stream/Repeat.cpp]: You dont need to make an output buffer, regular output buffer api. you can just only consume numElemsOut/_repeatCount and produce _repeatCount*(numElemsOut/_repeatCount)

Comment (ncorgan) [reply stream/Repeat.cpp]: Updated and rebased.

**Comment by ncorgan** [comment]
Looking at this, mute is more of an event block. Do you agree?

File: stream/Mute.cpp [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        const auto elemsBytes = elems * _dtypeSize;
+
+        auto input = this->input(0);
+        auto output = this->output(0);
+
+        const void* buffIn = input->buffer();
+        void* buffOut = output->buffer();
+
+        if(_mute) std::memset(buffOut, 0, elemsBytes);
+        else std::memcpy(buffOut, buffIn, elemsBytes);
```
Comment (guruofquality) [inline stream/Mute.cpp]: You could make this one of those passive forward blocks.

* when muted, do not forward messages, post zero buffer of equal size, same dtype
* when unmuted, forward messages and forward input buffer

I think gateway is a good example of this (minus that it doesnt mute) https://github.com/pothosware/PothosBlocks/blob/master/stream/Gateway.cpp

**Comment by guruofquality** [comment]
> Looking at this, mute is more of an event block. Do you agree?

seems like a stream block, kind of like gateway or copy

File: stream/Mute.cpp [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        if(0 == elems)
+        {
+            return;
+        }
+
+        auto inputPort = this->input(0);
+        auto outputPort = this->output(0);
+        Pothos::BufferChunk output;
+
+        if(_mute) output = Pothos::BufferChunk(_dtype, elems);
```
Comment (guruofquality) [inline stream/Mute.cpp]: I dont think this is guaranteed to be zeros

**Pull request merged**